src/Dialog/Dialog.jsx

```diff
@@ -6,7 +6,8 @@ import Form from 'react-jsonschema-form';
 import CustomSchemaField from './formComponents/CustomSchemaField/CustomSchemaField';
 import {fetchRelationFields, clearData} from './DialogActions';
 
-class GeneratedDialog extends Component {
+export class GeneratedDialog extends Component {
+// class GeneratedDialog extends Component {
```

Comment:
I don't know what do with this because in production we would like to have only exported class with connect but out test stack needs also exporting tested class.